src/system/Zikula/Module/SearchModule/AbstractSearchable.php

```diff
@@ -13,6 +13,7 @@
 
 namespace Zikula\Module\SearchModule;
 
+use Symfony\Component\DependencyInjection\ContainerInterface;
```

Comment:
An advantage of the current approach would be that methods in the abstract are available.

I'm not opposed to your idea, but I am unwilling to code it. If you are willing to do so, please do so.